pyEpiabm/pyEpiabm/core/person.py

```diff
@@ -47,6 +47,7 @@ def __init__(self, microcell):
         self.next_infection_status = None
         self.time_of_status_change = None
         self.infection_start_time = None
+        self.isolation_start_time = None
```

Comment:
person isolation_start_time is also used to determine whether or not the person is currently isolating (start_time= None -> not isolating). This usage of the attribute ought to be documented somewhere

Reply:
I definitely agree - I do still find this slightly non-intuitive so good to have it explained somewhere!

Reply:
This is now documented here: https://github.com/SABS-R3-Epidemiology/epiabm/wiki/Interventions